src/guide/typescript-support.md

```diff
@@ -133,6 +133,47 @@ const Component = defineComponent({
 })
 ```
 
+### Augmenting Types for Use with Plugins or User Definitions
+
+Vue 3 provides `globalProperties` property for [plugins](./plugins.html#writing-a-plugin) to add to Vue’s global properties in each component instance. In some cases, we also need to [do it by ourselves](../api/application-config.html#globalproperties):
+
+```typescript
+// User Definition
+import axios from 'axios'
+
+const app = Vue.createApp()
```

Comment:
```suggestion
const app = Vue.createApp({})
```

I believe an empty object has to be passed to avoid an error being thrown when the app is mounted.

Reply:
I have fixed them and left some thoughts, Thanks for your suggestions.